scripts/spin-up-new-devnet.sh

```diff
@@ -2,39 +2,67 @@
 parent_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
 cd "$parent_path/.."
 
+echo "Current CLUSTER_ENV is $CLUSTER_ENV"
+echo "Change this env to devnet|testnet to change cluster network"
+
 echo "Stopping demo in case anything is already running"
 ./scripts/stop-demo.sh
 
+source ./scripts/setup-envs.sh
+
 echo "Prepare devnet files"
-./scripts/reset-devnet.sh
+source ./scripts/reset-devnet.sh
 
 echo "Starting cardano node"
-docker-compose up cardano-node -d
+$COMPOSE_CMD up -d
+
+export CARDANO_NODE_SOCKET_PATH="${CLUSTER_WORKDIR}/node.socket"
 
 echo -n 'Waiting for the node socket ..'
-while ! [ -S devnet/node.socket ]
+while ! [ -S $CARDANO_NODE_SOCKET_PATH ]
 do
   echo -n "."
   sleep 0.1
 done
 echo '. done'
 
-echo "Setup cardano-cli env to connect to started network"
-export CARDANO_NODE_SOCKET_PATH=./devnet/node.socket
+echo "Changing node.socket owner to current user"
+sudo chown $USER:$USER "$CARDANO_NODE_SOCKET_PATH"
```

Comment:
This causes the following error on my system.

```
Changing node.socket owner to current user
chown: invalid group: ‘nazrhom:nazrhom’
```

Reply:
Something mac-specific I belive.